compiler/passes/convert-typed-uast.cpp

```diff
@@ -898,6 +898,13 @@ struct TConverter final : UastConverter,
                                 const resolution::CallInfo& ci,
                                 RV& rv);
 
+  // Primitive assignment between extern types
```

Comment:
This comment doesn't seem to line up with my intuition of `...`? This is the Chapel `...` right?